Ray_Serve/Ray-Serve.ipynb

```diff
@@ -0,0 +1,1897 @@
+    "history += response.split('|')[1:]\n",
+    "history"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d7c90bec-1b5e-4a1a-b67a-1e11f37169ea",
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 234
+    },
+    "id": "73855166-c226-40f9-a365-babb36f751b3",
+    "outputId": "17498373-04e4-4595-80af-88cb24c568f2",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "message = 'Je ne suis pas sûr.'\n",
+    "response = ray.get(endpoint_handle.get_response.remote(message, history))\n",
+    "response.split('|')[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5402694c-e4a3-4227-b438-0faab9dd9e3a",
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "25952d4a-43fe-49fa-a437-eb91813cde5c",
+    "outputId": "29e847c0-4819-46c6-d15c-681bab01e028",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "history += response.split('|')[1:]\n",
+    "history"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "78d25201-b26a-43c1-b8e7-0f6f5fc9fea8",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "At this point we have a service which can support the many functional and operational properties we expect to need in production, including scalability, separation of concerns, and composability."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2ac6bab4-7552-4ab5-9439-0df07d632fe4",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "serve.delete('multilingual_chat')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1c7f8816-e905-4c91-80f2-445a9d99dfa4",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">\n",
+    "    \n",
+    "__Lab activity: build a simpler translator__\n",
+    "    \n",
+    "You can use Google's Flan-T5 large model like this\n",
+    "    \n",
+    "```python\n",
+    "  from transformers import pipeline\n",
+    "\n",
+    "  pipe = pipeline(\"text2text-generation\", model=\"google/flan-t5-large\")\n",
+    "```\n",
+    "   \n",
+    "<br/>\n",
+    "    \n",
+    "This model can translate between English and several other languages if instructed to do so in the prompt. \n",
+    "    \n",
+    "Build a Serve application that\n",
+    "    \n",
+    "* detects in the input language\n",
+    "* translates (Engligh-Italian or the reverse) using Flan-T5\n",
+    "* returns the result\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf48a6a1-4f60-4ca7-889a-f31866cbe4c7",
+   "metadata": {
+    "id": "6eae86f3-34a4-4f59-87ba-df8d9a0d3928"
+   },
+   "source": [
+    "There is another, alternative pattern for composing applications out of services, which we can explore as well.\n",
+    "\n",
+    "## Declarative (graph) composition pattern\n",
+    "\n",
+    "What is the Deployment Graph API?\n",
```

Comment:
We actually want to avoid advertising this too much, we're pushing it down in our docs. This is in experimental stage

Reply:
Agreed; we may move this to the end or skip it for time reasons. 

I know at one point there was a question if we should add more content to this training but with Q&A and labs, it may well be too much for 3 hours already, so it's nice to have things we can skip or shorten!